dapp/src/components/ConnectWalletModal/ConnectWalletButton.tsx

```diff
@@ -127,6 +127,8 @@ export default function ConnectWalletButton({
       onError: (error: OrangeKitError) => {
         const errorData = orangeKit.parseOrangeKitConnectionError(error)
         setConnectionError(errorData)
+        if (connector.id === "orangekit-ledger-live")
+          logPromiseFailure(connector.disconnect())
```

Comment:
We could avoid this statement by handling the problem in the Orange Kit Ledger Live Provider inside the connect function. In the for loop, if `getAddress` fails with `UnsupportedBitcoinAddressError`, we could catch the error and continue the loop instead of failing the execution.

Reply:
e50d86d